fromapp/fromapp.php

```diff
@@ -49,7 +49,7 @@ function fromapp_settings(&$a, &$s)
 
 	/* Add our stylesheet to the page so we can make our settings look nice */
 
-	$a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . DI::baseUrl()->get() . '/addon/fromapp/fromapp.css' . '" media="all" />' . "\r\n";
+	DI::page()['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . DI::baseUrl()->get() . '/addon/fromapp/fromapp.css' . '" media="all" />' . "\r\n";
```

Comment:
I never saw this construct. How is it supposed to work?

Reply:
It's like:
```
$page = DI::page();
$page['htmlhead'] =  '...';
```

The `Friendica\App\Page` class implements `ArrayAccess` interface, that's why it works like an array. We could also explicit an add method :)

Reply:
I guess I would prefer some "get", "set" and "add" functions.

Reply:
Me too. Now its possible :)